packages/legend-application-studio/src/components/editor/edit-panel/uml-editor/StereotypeSelector.tsx

```diff
@@ -14,23 +14,34 @@
  * limitations under the License.
  */
 
-import { useState } from 'react';
+import { useCallback, useEffect, useRef, useState } from 'react';
 import { observer } from 'mobx-react-lite';
 import {
   CustomSelectorInput,
   createFilter,
   TimesIcon,
   ArrowCircleRightIcon,
+  VerticalDragHandleIcon,
 } from '@finos/legend-art';
 import { useEditorStore } from '../../EditorStoreProvider.js';
 import {
   type Profile,
   type StereotypeReference,
   type Stereotype,
   isStubbed_PackageableElement,
+  type AnnotatedElement,
 } from '@finos/legend-graph';
-import { stereotypeReference_setValue } from '../../../../stores/graphModifier/DomainGraphModifierHelper.js';
+import {
+  annotatedElement_arrangeStereotypes,
+  stereotypeReference_setValue,
+} from '../../../../stores/graphModifier/DomainGraphModifierHelper.js';
 import type { PackageableElementOption } from '@finos/legend-application';
+import {
+  type AllStereotypeDragSource,
+  CLASS_TAGGED_VALUE_DND_TYPE,
```

Comment:
why are we using `CLASS_TAGGED_VALUE_DND_TYPE` for `StereotypeSelector`? Also, let's rename these to `TAGGED_VALUE_DND_TYPE`

Let's also move that constant here.

Reply:
True!